src/index.js

```diff
@@ -1 +1,30 @@
-// Your server code here...
+// Import the express module
+import express from 'express';
+import bodyParser from 'body-parser';
+import mongoose from 'mongoose';
+
+// Create a new instance of express
+const app = express();
+
+const contactRoutes = require('../routes/contactRoutes');
+
+// Connect to Mongo database
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/contact-list');
+
+// Middleware
+app.use(bodyParser.json());
+app.use(contactRoutes);
+app.use((err, request, response) => response.status(500).send('Request error! ' + err));
```

Comment:
This error middleware won't work properly, need to supply 4 arguments, not just 3

```
(err, req, res, next)
```

Need to turn off the eslint rule for this block